internal/backends/linuxkms/Cargo.toml

```diff
@@ -31,10 +31,12 @@ i-slint-renderer-femtovg = { version = "=1.2.0", path = "../../renderers/femtovg
 
 [target.'cfg(target_os = "linux")'.dependencies]
 input = { version = "0.8.2" }
-xkbcommon = { version = "0.5.0" }
-calloop = { version = "0.10.5" }
-libseat = { version = "0.1.7" }
-nix = { version = "0.26.2" }
+xkbcommon = { version = "0.6.0" }
+calloop = { version = "0.11.0" }
+# nix broke semver that broke calloop https://github.com/nix-rust/nix/issues/2112
+pin_nix26 = { package="nix", version = "=0.26.2", default-features = false }
+libseat = { version = "0.2.1" }
+nix = { version = "0.27.0", features=["fs"] }
```

Comment:
This bring two version of nix.  (We had two before since calloop 0.10 used nix 0.25)
We could stay at nix 0.26 in this crate to avoid double dependencies in the release.
But we depends of nix through other crates anyway. Currently they all use 0.26 but it is possible that they update to nix 0.27 sooner or later.

But I think I actually prefer if we didn't have the double dependency so i'll downgrade

Reply:
But on the other hand from_bits_retain did not exist before

Reply:
My vote would be to keep the double dependency, as this is a small crate and that way when this is fixed, it's just a matter of removing a single line, right?